### PR TITLE
boundary_of_thermalization_zone_of_atomized_atoms_in_magnetron

### PR DESCRIPTION
**Pull request by Zulusssss**
add law and tests

### CODE REVIEW

File: symplyphysics/laws/chemistry/boundary_of_thermalization_zone_of_atomized_atoms_in_magnetron.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
+from sympy import Eq, solve
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output, dimensionless,)
+
+# Description
+## The atoms of the target material evaporate and move towards the substrate inside the magnetron. There is a distance at which the energy of the atomized
+## atom will decrease to the energy of the thermal motion of the atoms of the gas-discharge plasma. This distance is called the boundary of the thermalization zone.
```
Comment (blackyblack) [inline symplyphysics/laws/chemistry/boundary_of_thermalization_zone_of_atomized_atoms_in_magnetron.py:7]: Atomized atom sounds weird.

File: symplyphysics/laws/chemistry/boundary_of_thermalization_zone_of_atomized_atoms_in_magnetron.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+## The decrease in energy occurs due to the collision of the atomized atom with the gas atoms in the magnetron. The free path length is the distance that the sprayed
+## gas travels between two collisions.
+
+## Law is: L = N * l, where
+## L - boundary of thermalization zone of atomized atoms in magnetron,
+## N - the number of collisions of atomized atom with gas atoms (is a statistical quantity, can be neither an integer),
```
Comment (blackyblack) [inline symplyphysics/laws/chemistry/boundary_of_thermalization_zone_of_atomized_atoms_in_magnetron.py:13]: It depends on time and distance traveled by atom. Can we define some bounds, where this parameter is estimated? For example stationary atom would get infinite amount of collisions at infinite time period.

File: symplyphysics/laws/chemistry/boundary_of_thermalization_zone_of_atomized_atoms_in_magnetron.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+## l - free path length of atomized atom.
+
+boundary_of_thermalization_zone = Symbol("boundary_of_thermalization_zone", units.length)
+
+number_of_collisions_of_atoms = Symbol("number_of_collisions_of_atoms", dimensionless)
```
Comment (blackyblack) [inline symplyphysics/laws/chemistry/boundary_of_thermalization_zone_of_atomized_atoms_in_magnetron.py:18]: Of one atom or many atoms?

File: symplyphysics/laws/chemistry/boundary_of_thermalization_zone_of_atomized_atoms_in_magnetron.py (hunk 1 above)
Comment (Zulusssss) [reply symplyphysics/laws/chemistry/boundary_of_thermalization_zone_of_atomized_atoms_in_magnetron.py:7]: > Atomized atom sounds weird.

What's the best way? sprayed atom? pulverized atom?

**Review by blackyblack — APPROVED** [review]

**Pull request merged**